frontend/src/app/organizations/organizations-list/organizations-list.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <div class="col-lg-12">
+    <h2>Lista organizacji</h2>
+  </div>
+</div>
+<div class="row">
+  <div class="col-lg-12">
+    <ng-container *ngIf="organizations">
```

Comment:
WIll not it be sufficient to instead of
```
<ng-container *ngIf="organizations">
<table *ngIf="organizations.length; else noOrganizations" class="table table-striped">
```
have here
```
<table *ngIf="organizations?.length; else noOrganizations" class="table table-striped">
```